### DISCUSSION TITLE
disable hover effect & select for specific column

### DISCUSSION THREAD

**Discussion by kmurari103** [iTwin/iTwinUI-react#461] [Q&A] [answered]
Hi there,
One of the columns in my table contains buttons for the user to interact with. I do not want the clicking of these buttons to trigger the select/click row event, but clicking any other column should.  
Same with hover effect- I do not want hovering over these buttons to show the Tooltip but other column should.(See image below)

![image](https://user-images.githubusercontent.com/27068008/144832367-53bb09e3-1c3e-49a7-b6c5-25d6f3346949.png)

**Comment by bentleyvk** [comment, marked as answer]
Hi,
We have this example in Table selection story: https://itwin.github.io/iTwinUI-react/?path=/story/core-table--selectable
![image](https://user-images.githubusercontent.com/36186912/144832861-3007f7c9-4c45-404d-86ed-ae64968cd184.png)
And to handle Tooltip you will also need to:
```tsx
  <a
    // ...
    onMouseEnter={(e) => e.stopPropagation()}
  >
    {...}
  </a>
```
